src/OSPSuite.R/Services/DataRepositoryTask.cs

```diff
@@ -39,6 +41,15 @@ public interface IDataRepositoryTask
       /// <returns></returns>
       DataColumn AddErrorColumn(DataColumn column, string name, string errorType);
 
+      /// <summary>
+      ///    Removes the <paramref name="column" /> from the <paramref name="dataRepository" />. if the column is a related
+      ///    column, the association will also be removed. If the column is a base grid column, it can only be removed if the
+      ///    data repository has no other column
+      /// </summary>
+      /// <param name="dataRepository">DataRepository containing the column to remove</param>
+      /// <param name="column">Column to remove</param>
+      void RemoveColumn(DataRepository dataRepository, DataColumn column);
```

Comment:
@PavelBal This is the method to use to remove a column from a data repository

Reply:
Love the comments!